### DISCUSSION TITLE
BOT send PDF base64 API to VISITOR

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1843] [General]
Using CALLS REST API, querying my API I get base64 to generate a PDF

Can I use this return for the BOT to send a file to VISITOR?

The PDF would be the Invoice.

Or even save the base64
/file/downloadfile/

for the BOT to send to the VISITOR?

Or is there something simpler to do this on the LHC and I don't know how to send the BOT to the VISITOR

**Comment by remdex** [comment]
Hi,

At the moment it's not possible directly from API call save it as a file. As always possible to develop.

But you can 
* Then lhc makes request to your API. You make Rest API call and save file using https://api.livehelperchat.com/#/file/post_restapi_file afterward your API just returns text message following https://doc.livehelperchat.com/docs/chat/bbcode file formating syntax.
* Perhaps would be possible to chain these requests directly in LHC, never tried :)
